src/index.js

```diff
@@ -69,10 +69,6 @@ exports.encode = function encode (digest, code, length) {
     throw new Error('digest length should be equal to specified length.')
   }
 
-  if (length > 127) {
```

Comment:
Correction, in a full multihash buffer there is a:

`<multicodec varint><varint digest length><digest>`

See https://github.com/multiformats/multihash#format for reference